docs-src/go/how-to-list-workflow-executions-using-the-client-in-go.md

```diff
@@ -9,4 +9,26 @@ tags:
   - client
 ---
 
-Use [`Client.ListWorkflow`](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow).
+The [`ListWorkflow()`](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow) function retrieves a list of [Workflow Executions](/concepts/what-is-a-workflow-execution) that match the [Search Attributes](/concepts/what-is-a-search-attribute) of a given [List Filter](/concepts/what-is-a-list-filter).
+The metadata returned from the [Visibility store](/concepts/what-is-visibility) can be used to get a Workflow Execution's history and details from the [Persistence store](/concepts/what-is-a-temporal-cluster#persistence).
+
+Use a List Filter to define a `request` to pass into `ListWorkflow()`.
+
+```go
+request := "CloseTime = missing"
```

Comment:
`request` has to be a full `ListWorkflowExecutionsRequest` object here, not just a string, so something like:

```suggestion
request := &workflowservice.ListWorkflowExecutionsRequest{ Query: "CloseTime = missing" }
```